docs/naps/6-contributable-menus.md

```diff
@@ -217,51 +233,84 @@ It may never make sense for such plugins to distribute their widgets across the
 data formats or layer types.
 
 We therefore give plugin developers full control over their own submenu under `Plugins->My Plugin`.
-Plugin developers can organize all contributions under this submenu as they see fit, including adding their own submenus of arbitrary depth.
+Plugin developers can organize all contributions under this submenu as they see fit, including adding 
+their own submenus of arbitrary depth.
+
+All commands offered by a plugin will always be available in the plugin's submenu, regardless of
+their presence in other menus. If the plugin defines its own submenus, these will always be first,
+followed by a separator, then any commands with no dedicated menu entry for the plugin's own 
+submenu. These will be alphabetically ordered.
```

Comment:
I think I forgot this detail. Are we sure we want to duplicate menu items (i.e., place contribution in the plugin's submenu *and* in another menu, defined by the plugin?)?

Reply:
No, we are not sure, but personally I think it is better to have a way to see all of a plugin's contributions in one place. But we are definitely not sure, this is 100% open to discussion.